langgraph/graph/state.py

```diff
@@ -269,7 +269,7 @@ def attach_edge(self, starts: Union[str, Sequence[str]], end: str) -> None:
             elif end != END:
                 # subscribe to start channel
                 self.nodes[end].triggers.append(starts)
-        else:
+        elif end != END:
```

Comment:
No Else condition now?

Reply:
else would be if end is END, which we don't need to do anything for